apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -87,10 +135,11 @@ WarningLabel.propTypes = {
  * Given a search query, generate a results list of animationProps objects that
  * can be displayed and used to add an animation to the project.
  * @param {string} searchQuery - text entered by the user to find an animation
+ * @param {string} categoryQuery - name of category user selected to filter animations
  * @return {Array.<SerializedAnimationProps>} - Limited list of animations
  *         from the library that match the search query.
  */
-function searchAnimations(searchQuery) {
+function searchAnimations(searchQuery, categoryQuery) {
```

Comment:
I think there will be continuous improvement on this function, particularly to help make animations easier to find, but I'm definitely open to ideas about making this fasters. Right now I'm just getting search results and category results and then taking the intersection.